echopype/echodata/combine.py

```diff
@@ -30,6 +30,47 @@ def assemble_combined_provenance(input_paths):
     )
 
 
+def check_and_correct_reversed_time(combined_group, old_time, new_time, time_str, sonar_model):
```

Comment:
Really nice and clean way of generalizing the previous functionality!

Reply:
Thank you!